plenum/test/test_bootstrapping.py

```diff
@@ -49,6 +47,5 @@ def testConnectWithoutKeySharingFails(tdir_for_func):
         with TestNodeSet(names=nodeNames, tmpdir=tdir_for_func,
                          keyshare=False) as nodes:
             with Looper(nodes) as looper:
-                    looper.run(
-                            checkNodesConnected(nodes, NOT_CONNECTED))
+                looper.runFor(2)
```

Comment:
This just need to check if a particular exception is raised when nodes try to connect to others which happens next after nodes creation, that is why the timeout.